lib/Loader.php

```diff
@@ -272,8 +276,8 @@ public function get_loader() {
 				}, $path_locations );
 			} else {
 				Helper::deprecated(
-					'add_filter( \'timber/loader/paths\', [\'path/to/my/templates\'] ) in a non-associative array',
-					'add_filter( \'timber/loader/paths\', [ 0 => [ \'path/to/my/templates\' ] ] )',
+					'add_filter( \'timber/locations\', [\'path/to/my/templates\'] ) in a non-associative array',
+					'add_filter( \'timber/locations\', [ 0 => [ \'path/to/my/templates\' ] ] )',
```

Comment:
If I understand this deprecation notice correctly — it's warning that the arguments for `timber/locations` has changed. While this is true for `timber/loader/paths`, I don't believe it's needed

Reply:
True, and possibly related to the tests still failing. Still catching up on how the tests check for deprecations.